cdr_schemas/prospectivity_input.py

```diff
@@ -58,6 +59,17 @@ class ProcessedDataLayer(BaseModel):
     normalization_method: str  # source: LayerDataType
 
 
+class CriticalMineralAssessment(BaseModel):
+    crs: str
+    extent: Polygon
```

Comment:
https://github.com/DARPA-CRITICALMAAS/cdr_schemas/blob/main/cdr_schemas/features/polygon_features.py#L13

We already have a `Polygon` schema that is essentially the same that I can tell.  Is there enough value in this library to justify adding a third-party dependency?

Reply:
They have implemented robust conversion to WKT coordinates:

https://github.com/developmentseed/geojson-pydantic/blob/4c85ed5b54bdcca9ae75872b57b0e646cb513b27/geojson_pydantic/geometries.py#L91

Reply:
I guess that is fine but the cdr_schema is really just a definition of the wire protocol.  We don't really have any behavior on these objects, they just get persisted.

Reply:
fyi we decided to change this to `geojson_pydantic.MultiPolygon` because a CMA might not be of one contiguous area. It looks like your `PolygonFeatures` class might perform a similar function, but it may be too specific to your needs regarding map feature extraction (as opposed to our needs related to running TA3 models)